x-pack/plugins/ml/public/application/components/data_grid/common.ts

```diff
@@ -205,6 +205,7 @@ export const getDataGridSchemaFromESFieldType = (
       break;
     // keep schema undefined for text based columns
     case ES_FIELD_TYPES.KEYWORD:
+    case ES_FIELD_TYPES.VERSION:
```

Comment:
This change should have no effect, as this last collection of cases essentially behave like a `default`